web_developer_mode_optional/static/description/icon.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I can not confirm that it is an OCA policy, but I can say that it's the first time I see a vectorial icon in OCA modules. (But I don't know all of them)
@jgrandguillaume, @gurneyalex : What'is your PoV about that point ?

My *personal* PoV is : 

* change icon is not a real need ;
* add svg file make PR on github and git diff less readable ;

But it's not a blocking point for me.